assemblies/evaluating-rag-systems.adoc

```diff
@@ -0,0 +1,103 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+[id="evaluating-rag-systems-with-Ragas_{context}"]
+= Evaluating RAG systems with Ragas
+
+ifndef::upstream[]
+[IMPORTANT]
+====
+ifdef::self-managed[]
+Retrieval-Augmented Generation Assessment (Ragas) is currently available in {productname-long} as a Technology Preview feature.
+endif::[]
+ifdef::cloud-service[]
+Retrieval-Augmented Generation Assessment (Ragas) is currently available in {productname-long} as a Technology Preview feature.
+endif::[]
+Technology Preview features are not supported with {org-name} production service level agreements (SLAs) and might not be functionally complete.
+{org-name} does not recommend using them in production.
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of {org-name} Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+endif::[]
+
+[role='_abstract']
+As an AI engineer, you can use link:https://docs.ragas.io/en/stable/[Ragas] (Retrieval-Augmented Generation Assessment) to measure and improve the quality of your RAG systems in {productname-short}. Ragas provides objective metrics that assess retrieval quality, answer relevance, and factual consistency, enabling you to identify issues, optimize configurations, and establish automated quality gates in your development workflows.
+
+Ragas is integrated with {productname-short} through the Llama Stack evaluation API and supports two deployment modes: an inline provider for development and testing, and a remote provider for production-scale evaluations using {productname-short} pipelines.
+
+== About Ragas evaluation
+
+Ragas addresses the unique challenges of evaluating RAG systems by providing metrics that assess both the retrieval and generation components of your application. Unlike traditional language model evaluation that focuses solely on output quality, Ragas evaluates how well your system retrieves relevant context and generates responses grounded in that context.
+
+=== Key Ragas metrics
+
+Ragas provides link:https://docs.ragas.io/en/stable/concepts/metrics/available_metrics/[multiple metrics] for evaluating RAG systems. Here are some of the metrics:
+
+*Faithfulness*:: Measures how factually consistent the generated answer is with the retrieved context. A high faithfulness score indicates that the answer is well-grounded in the source documents, reducing the risk of hallucinations. This is critical for enterprise and regulated environments where accuracy and trustworthiness are paramount.
```

Comment:
first phrase is awkward to parse: "Measures how factually consistent the generated answer **is with** the retrieved context."
Consider:
Measures the generated answer to determine whether it is consistent with the retrieved context.